config.toml

```diff
@@ -206,6 +206,12 @@ parent = "Project"
 url = "/roadmap"
 weight = 50
 
+[[menus.main]]
```

Comment:
This shouldn't be required - we already list the other Security doc under "Project".

Reply:
It wasn't there before, but believe this was just an oversight. Fixed the link 👍